release-notes/24.05/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>CheriBSD 24.05 Release Notes</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="/assets/dist/css/bootstrap.min.css" rel="stylesheet">
+
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <meta name="msapplication-TileColor" content="#b91d47">
+  </head>
+  <body>
+    <main role="main">
+      <section class="jumbotron text-left rounded-0 m-0">
+  <div class="container">
+<h1>CheriBSD 24.05 Release Notes</h1>
+<h2>Overview</h2>
+<p>
+Welcome to the Spring 2024 (24.05) release of the CheriBSD operating system.  As well as a general update of the baseline FreeBSD OS (corresponding to FreeBSD CURRENT as of January 2024) from which CheriBSD is derived, we have introduced several new research components which will be of interest to the CHERI and CheriBSD community:
+</p>
+<ul>
+<li>Library Compartmentalization (<a href="https://man.cheribsd.org/cgi-bin/man.cgi/release-24.05/c18n">https://man.cheribsd.org/cgi-bin/man.cgi/release-24.05/c18n</a>) is increasingly mature.  It is possible to enabled it system wide by setting the security.cheri.lib_based_c18n_default sysctl to 1.  While this implementation is sufficently functional to run the full desktop environment discussed below, it is neither complete nor intended to be secure.
```

Comment:
enabled -> enable

sufficiently functional -> functional enough

Reply:
Why the latter? (Though I now note s/sufficently/sufficiently/)

Reply:
Reads more clearly, at least to me, the first is awkward, but maybe we're doing the UK/US passive/active thing.

Reply:
I've reworded this section in the course of adding a mention of `procstat c18n`.